packages/libp2p-client/src/client.ts

```diff
@@ -75,11 +71,16 @@ export class AgentLibp2pClient implements IAgentPlugin {
               // console.log("msg of source: ", msg)
               message = message + (msg.toString().replace('\n',''))
             }
-            context?.agent.handleMessage({ raw: message })
+            context?.agent.handleMessage({ raw: message, save: true })
```

Comment:
we're trying to deprecate the `save` param and encourage users to deliberately save messages that make sense for them after handling(parsing) by calling methods of the data-store